lib/utils/docker.coffee

```diff
@@ -48,6 +48,12 @@ exports.appendOptions = (opts) ->
 			alias: 't'
 		},
 		{
+			signature: 'arg'
+			parameter: 'arg'
+			description: 'Set a build-time variable (eg. "-e \'ARG=value\'"). Can be specified multiple times.'
+			alias: 'e'
```

Comment:
We do use `--env/-e` in `local push` and it's a very similar option, I just went by Docker's convention of calling them "Build Args": https://docs.docker.com/engine/reference/commandline/build/